modules/ipi-install-setting-cluster-node-hostnames-dhcp.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+//  *installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ipi-install-setting-cluster-node-hostnames-dhcp_{context}"]
+= Setting the cluster node hostnames through DHCP
+
+On {op-system-first} machines, the hostname is set through NetworkManager. 
+By default, the machines obtain their hostname through DHCP. 
+If the hostname is not provided by DHCP, set statically through kernel arguments, or another method, it is obtained through a reverse DNS lookup. 
+Reverse DNS lookup occurs after the network has been initialized on a node and can take time to resolve. Other system services can start prior to this and detect the hostname as `localhost` or similar. 
+You can avoid this by using DHCP to provide the hostname for each cluster node.
+
+Additionally, setting the hostnames through DHCP can bypass any manual DNS record name configuration errors in environments that have a DNS split-horizon implementation.
```

Comment:
```suggestion
You can avoid this delay in assigning hostnames by using DHCP to provide the hostname for each cluster node. Additionally, setting the hostnames through DHCP can bypass any manual DNS record name configuration errors in environments that have a DNS split-horizon implementation.
```

Reply:
good suggestion. Thanks for the review @JoeAldinger